### ISSUE TITLE
The value is overriden and can not be changed

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#1683]
@childsb 

In both Preview 4 and latest code:
- Virgin setup
- New HTML File
- Add ContentPane
- Add H2 inside of ContentPane
- In Prop palette, set text-align to center and font-weight to bold
- Click on "All" link at top of Prop palette

Result=> Alert that says "The value is overriden and can not be changed"

**Comment by JonFerraiolo** [comment]
Still investigating. What I've learned so far is that the problem in this case has to do with:
- 'font' property is registering the callback that causes the alert to appear
- Claro's document.css includes a style rule with selector `.claro H2` as follows:

```
.claro h2 {
  font-size: 1.364em;
  line-height: 1.455em;
  margin-top: 1em;
  margin-bottom: 0.60em;
  font-weight: normal;
}
```

Probably the overrides logic has a major bug that we just haven't noticed yet.

**Comment by JonFerraiolo** [comment]
Closing, at least for time being. This was fixed "by accident" because I commented out some of the overrides logic from Cascade.js as part of 9c3ae7968c as part of my fixes to add property array logic to things that interact with States.js. (For the commented out code, look for this comment in Cascade.js: `/*Disabled hasOverride logic - had bugs, causes problems with logic and not sure it helps user`) 

I feel it's OK to close this because if we resurrect the overrides logic, it will probably work somewhat differently when resurrected.
